tool/gravity/cli/rpcagent.go

```diff
@@ -476,15 +476,16 @@ func collectAgentStatus(env *localenv.LocalEnvironment) (statusList rpc.StatusLi
 func verifyOrDeployAgents(env *localenv.LocalEnvironment) error {
 	statusList, err := collectAgentStatus(env)
 	if err != nil {
+		env.Println("Failed to verify agents. Ensure gravity-site and etcd are up and running.")
```

Comment:
"Couldn't verify upgrade agents status. If some are offline, they won't be redeployed automatically".

Also, can we make this a colored warning e.g. yellow? And as a small nit, I don't think we add dots to the end of our progress messages.